### ISSUE TITLE
Question on the exercises

### ISSUE THREAD

**Issue by shaw1903** [markjprice/cs9dotnet5#18]
Hello,

I've really enjoyed the book so far but as i am new to C# i can't seem to find a solution to exercise 15.3 and have struggled through some previous exercises. 

Are there practical examples of the additional exercises anywhere?

**Comment by markjprice** [comment]
Hi,

For Exercise 15.3, which is about re-implementing earlier exercises that were Console Apps as Razor Pages, I did not provide solutions since there already solutions implemented as Console Apps. But I agree it would be better if I did provide Razor Page solutions too. I'm super busy with final drafts of the sixth edition at the moment. I will try to make time to do this but it might not be for a couple of months so I will leave this issue open for now to remind me.

For those who do not know, the chapter projects contain exercise solutions as well as step-by-step task solutions. Answers to all end-of-chapter questions are in the Appendix downloadable from here: 
https://static.packt-cdn.com/downloads/9781800568105_Appendices.pdf
